openssl/src/provider.rs

```diff
@@ -55,6 +55,10 @@ impl Provider {
                 retain_fallbacks as _,
             ))?;
 
+            // OSSL_PROVIDER_try_load seems to leave errors on the stack, even
+            // when it succeeds.
+            let _ = ErrorStack::get();
```

Comment:
I assume this is a regression in 3.2? We should probably file a bug upstream if so.

Reply:
I'm not sure what changed